production/operator/crds/monitoring.grafana.com_grafanaagents.yaml

```diff
@@ -4275,6 +4275,94 @@ spec:
                             if specified. The name is used in metrics and logging
                             in order to differentiate queues.
                           type: string
+                        oauth2:
+                          description: Oauth2 for URL
+                          properties:
+                            clientId:
+                              description: The secret or configmap containing the
+                                OAuth2 client id
+                              properties:
+                                configMap:
+                                  description: ConfigMap containing data to use for
+                                    the targets.
+                                  properties:
+                                    key:
+                                      description: The key to select.
+                                      type: string
+                                    name:
+                                      description: 'Name of the referent. More info:
+                                        https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+                                        TODO: Add other useful fields. apiVersion,
```

Comment:
Do we want to leave this TODO in the CRD? maybe we want to get rid of it before

Reply:
This TODO was automatically generated from upstream. I did not touch the CRDs manually, therefore I would submit it as is.

Reply:
Ah sorry, didn't see that it's comming from upstream, never mind